### ISSUE TITLE
no mention of tabs in `:h local-options`

### ISSUE THREAD

**Issue by nhooyr** [vim/vim#976]
`:help local-options` says "When editing a buffer that has been edited before, the last used window options are used again.”
This only applies if the buffer has been edited before in the current tab.

E.g.

``` vim
set nu
tabe foo
setl nonu
tabp
sp foo
```

`foo` will have line numbers enabled because the last used window options are not used again as the window was in another tab.

**Comment by brammool** [comment]
Amol Sethi wrote:

> `:help local-options` says "When editing a buffer that has been edited
> before, the last used window options are used again.” This only
> applies if the buffer has been edited before in the current tab.
> 
> E.g.
> 
> ``` vim
> set nu
> tabe foo
> setl nonu
> tabp
> sp foo
> ```
> 
> `foo` will have line numbers enabled because the last used window
> options are not used again as the window was in another tab. 

This is not related to tab pages, it's because the window where 'number'
was reset wasn't closed.  This does the same:

```
set nu
sp foo
setl nonu
windcmd w
sp foo
```

I'm not sure if we should change this behavior, it's been like this for
a very long time.

## 

hundred-and-one symptoms of being an internet addict:
9. All your daydreaming is preoccupied with getting a faster connection to the
   net: 28.8...ISDN...cable modem...T1...T3.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\
\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by nhooyr** [comment]
What you describe should be in `'local-options'` then as well, it doesn't mention anything about open windows.

 Nevertheless, how come the same happens for

``` vim
set nu
tabe foo
setl nonu
close
sp foo
```

Here the window was closed but when opening foo again, line numbers were still enabled.

**Comment by lacygoill** [comment]
I think the issue mentioned in the OP has been fixed by 25782a7ff4755daf16c2e1cb5e5f826b13b672ce.
And the one in the last post has been fixed by e59215c7dcae17b03daf39517560cfaa03314f5a.
